package.json

```diff
@@ -27,12 +27,14 @@
     "@angular/router": "4.1.3",
     "@types/clipboard": "1.5.34",
     "angular-datatables": "4.1.1",
-    "angular2-highlight-js": "5.0.1",
+    "angular2-highlight-js": "^5.0.1",
```

Comment:
I see that angular2-highlight-js is embedded in our repo, this shouldn't be necessary anymore.